content/bookmarks.json

```diff
@@ -1,28 +1,32 @@
 [
     {
         "favicon": "image://theme/icon-m-service-jolla",
-        "title": "User Guide",
-        "url": "http://jolla.com/guide/"
+        "title": "Jolla",
+        "url": "http://jolla.com/"
     },
     {
         "favicon": "image://theme/icon-m-service-jolla",
-        "title": "How-to tutorial videos",
-        "url": "http://m.youtube.com/playlist?list=PLQgR2jhO_J0y8YSSvVd-Mg9LM88W0aIpD&desktop_uri=%2Fplaylist%3Flist%3DPLQgR2jh
-O_J0y8YSSvVd-Mg9LM88W0aIpD"
+        "title": "How-to videos",
+        "url": "http://m.youtube.com/playlist?list=PLQgR2jhO_J0y8YSSvVd-Mg9LM88W0aIpD&desktop_uri=%2Fplaylist%3Flist%3DPLQgR2jhO_J0y8YSSvVd-Mg9LM88W0aIpD"
```

Comment:
Should be:
https://www.youtube.com/playlist?list=PLQgR2jhO_J0zrl6kLWusekAFr9aHR9jcH